failing-byon-udp-test.yaml

```diff
@@ -0,0 +1,50 @@
+name: byon test
+services:
+- type: org.apache.brooklyn.ports.client
+  location:
+    byon:
+      hosts:
+      - ssh: aws-host-name1
+        user: centos
+        privateKeyFile: ~/.ssh/id_rsa
+- type: org.apache.brooklyn.entity.server.udp.security.groups
+  location:
+    byon:
+      hosts:
+      - ssh: aws-host-name2
+        user: centos
+        privateKeyFile: ~/.ssh/id_rsa
+- type: test-case
+  brooklyn.children:
+  - type: test-case
+    brooklyn.children:
+    - type: assert-up-and-running-initial
+      targetId: entity.ports.udp.server
```

Comment:
The test structure here looks wrong - `targetId: entity.ports.udp.server` means do this test against the entity whose id is `entity.ports.udp.server` - but there is no such entity here.